pkg/cmd/context.go

```diff
@@ -116,6 +116,10 @@ func NewContextCommand(ctx context.Context) *cobra.Command {
 
 			for k, v := range conf.APIs {
 				fmt.Fprintf(os.Stdout, "%s\n", string(k))
+				if len(v.Contexts) == 0 {
+					fmt.Fprint(os.Stdout, "\t- no tenants yet, currently not usable\n")
```

Comment:
Out of curiosity, why not just use `fmt.Print`? :thinking:

Reply:
I plan to add tests for the commands here, in which case it would be useful to just pass an `io.Writer` to these functions and use it to compare outputs. 🙂

Reply:
Understood :+1: Then let's keep it if there's a near term plan to rely on it.